apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
```

Comment:
Yeah, I didn't wanted it related to a field in the error messages so that it could be shown eventually at the top of the form in the ``__all__`` section.